pkgs/tools/networking/nikto/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, fetchFromGitHub
+, perlPackages
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nikto";
+  version = "2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "sullo";
+    repo = "nikto";
+    rev = "c83d0461edd75c02677dea53da2896644f35ecab";
```

Comment:
Can you request tags upstream? This makes updating much easier.

Reply:
There is already an [open issue](https://github.com/sullo/nikto/issues/590) for that.